Linter package added and basic files created

TadesseDev opened the pull request:
## new files
- index.html
- main.css
- test.md
## readme templet modified. 
- readme.md file content modified according to the project content.
## other 
- test.md file created and ignored with the .ignore file

Commit:
Include Linter Package

Commit:
Create index.html file

Commit:
Create main.css file

Commit:
Ignore the test.md file

gemmen29 commented on .gitignore:
```diff
@@ -0,0 +1 @@
+test.md
```
Please add `node_modules` in the .gitignore file because it's a requirement in this project.

TadesseDev replied:
Hello @gemmen29, Thanks for the review. Altho, are the [.hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc) and [.stylelintrc.json](https://github.com/microverseinc/linters-config/blob/master/html-css/.stylelintrc.json) files suppose to be in the remote repo or its enough if those are in local machine. I thought since those are in [.gitignore](https://github.com/Tadesse-Alemayehu/Hello-world/blob/linter-and-basic-files/.gitignore) file they wouldn't be required here and I delete them.

gemmen29 reviewed (changes requested):
Hi @Tadesse-Alemayehu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- Kindly add [.hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc) and [.stylelintrc.json](https://github.com/microverseinc/linters-config/blob/master/html-css/.stylelintrc.json) files in linters repo.
_Check the comments under the review._

![](https://media.giphy.com/media/tKyrc0NTlEcGWdThp4/giphy.gif)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Add .hintrc and .stylelintrc.json files

Commit:
More files to be ignored. 

## Ignored files.
-- node_modules/
-- package-lock.json
-- package.json
-- .hintrc
-- .stylelintrc.json

Commit:
remove the  .stylelintrc.json from remote repo.

Commit:
Remove the .hintrc from rote repos.

Donard97 reviewed (changes requested):
## Hi @Tadesse-Alemayehu, ✋  
    
Great work on making the changes requested by a previous reviewer 👏 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
    
## Required Changes ♻️
- [ ] Please add .stylelintrc.json and .hintrc following linters-config [link](https://github.com/microverseinc/linters-config/tree/master/html-css). They should be added to the root directory. 💪 
    
## Optional suggestions
    
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
    
**Cheers and Happy coding!👏👏👏**
    
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
    
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
    
 ------
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
    ```

Tadesse-Alemayehu commented:
Hello @Donard97, thank you for the review, am I required to configure a linter on my local machine for those file to be present or adding the two files to the root of my GitHub repo will do it.

Donard97 commented:
> Hello @Donard97, thank you for the review, am I required to configure a linter on my local machine for those file to be present or adding the two files to the root of my GitHub repo will do it.

Hello @Tadesse-Alemayehu, kindly add files of stylelint and hint in your root and push them in your remote repo. Keep coding!  💪

Commit:
Add .hintrc and .stylelintrc.json files

Commit:
Modify .hintrc and .stylelintrc.json

Donard97 reviewed (approved):
# Hi @Tadesse-Alemayehu ,  ✋ 

**Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:** 

## Congratulations! 🎉

![giphy (6)](https://user-images.githubusercontent.com/74506933/153270009-73e6bc4c-05b2-4f3a-a697-bacd22a64e2e.gif)

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
